plugins/mcreator-core/jsontriggers/item_predicate_damage_full.json

```diff
@@ -0,0 +1,49 @@
+{
+  "args0": [
+    {
+      "type": "field_number",
+      "name": "min",
+      "value": 1,
+      "precision": 1,
+      "min": 1
+    },
+    {
+      "type": "field_number",
+      "name": "max",
+      "value": 100,
+      "precision": 1
+    },
+    {
+      "type": "input_dummy"
+    },
+    {
+      "type": "field_number",
+      "name": "min2",
+      "value": 1,
+      "precision": 1,
+      "min": 1
+    },
+    {
+      "type": "field_number",
+      "name": "max2",
+      "value": 2000,
+      "precision": 1
+    }
+  ],
+  "inputsInline": false,
+  "output": "ItemPredicateEntry",
+  "colour": 130,
+  "extensions": [
+    "double_min_max_fields_validator"
+  ],
+  "mcreator": {
+    "group": "damage",
+    "toolbox_id": "components",

```

Comment:
It is quite hard to know what can be put together and what not from this colorful toolbox, I struggled until I got it right:

<img width="1130" height="747" alt="image" src="https://github.com/user-attachments/assets/86edc94b-0f1e-4b39-9c8d-2197be210ceb" />

I advise you to move item comeponents (and item predicate) stuff into another toolbox or sub-toolbox. You may also want to attach a predicate by default to let user know what should be connected there.